CMakeLists.txt

```diff
@@ -167,10 +167,12 @@ endfunction(FISH_LINK_DEPS_AND_SIGN)
 
 # Define libfish.a.
 add_library(fishlib STATIC ${FISH_SRCS} ${FISH_BUILTIN_SRCS})
+execute_process(COMMAND sh "-c" "find /usr/lib/libkvm.a 2> /dev/null" OUTPUT_VARIABLE KVM_LIBRARY)
+string(REPLACE "\n" "" KVM_LIB "${KVM_LIBRARY}")
```

Comment:
I am not enthused by this solution, this isn't typically how you use cmake.

I'd have to look up what the correct way is, but this appears very hacky.